lib/stop_watch_timer.dart

```diff
@@ -44,6 +44,8 @@ class StopWatchTimer {
 
     if (mode == StopWatchMode.countDown) {
       final value = presetMillisecond;
+      _second = getRawSecond(value);
+      _minute = getRawMinute(value);
       _rawTimeController = BehaviorSubject<int>.seeded(value);
```

Comment:
Here, _second and _minute is initialized for coundDown, given that the initial value of 0 is not necessarily correct.